docs/reference/remap/replace.cue

```diff
@@ -20,6 +20,14 @@ remap: functions: replace: {
 			required:    true
 			type: ["string"]
 		},
+		{
+			name:        "count"
+			description: "The maximum number of replacements to perform. -1 means replace all matches."
+			required:    false
+			default:     -1
+			type: ["integer"]
+
+		},
 	]
 	return: ["string"]
```

Comment:
I was noticing the return values. Any reason this is documented as a list?

Reply:
Yes, some functions can return different types depending on the input. For example `flatten` will return an `array` if the input is an `array` and a `map` if the input is a `map`.

Reply:
Sure, so shouldn't this just be `return: "string"`?

Reply:
This could be my lack of understanding `cue` then.. The return field is defined as:

https://github.com/timberio/vector/blob/94efee7c92b5fb208dbf39214ddd997b07226764/docs/reference.cue#L892

Is there a way to specify it so that for some functions the return field is a single value and other functions this return field is an array?

Reply:
Ping @binarylogic

Reply:
Yes, but we try not to do that since GraphQL typing doesn't like dynamic types, so let's leave as an array.